hibernate-core/src/main/java/org/hibernate/persister/entity/AbstractEntityPersister.java

```diff
@@ -1721,7 +1722,7 @@ public String generateSelectVersionString() {
 		SimpleSelect select = new SimpleSelect( getFactory().getDialect() )
 				.setTableName( getVersionedTableName() );
 		if ( isVersioned() ) {
-			select.addColumn( versionColumnName );
+			select.addColumn( getVersionColumnName(), VERSION_COLUMN_ALIAS );
```

Comment:
Is there possibility that this magical alias might conflict with existing alias? For instance, the existing alias created by session.createNativeQuery(). If so we might end up with new subtle bug when we closed this jira ticket.
Wild guessing and feel free to ignore my comment.